docs-sandbox/docs/03-install/index.mdx

```diff
@@ -0,0 +1,300 @@
+---
+title: Install
```

Comment:
I personally think this should be more on the lines of "Development Environment Configuration" - as "Install" is a bit of a misnomer for what you are doing. Did I miss where this was discussed and decided against?

Reply:
Sure 

> > cc @NukeManDan : I know you had come up with a plan to organize the installation content: I thought I'd port over what we have and you can modify it from there.
> 
> Yeah I had some opinions here: #514 I separated out the various bits in there except the migration /tweaking of pages content. Happy to make a new PR if you generally agree with the flow that had introduced?

All your suggestions seem valid to me. Why don't we merge this and you create a PR that focuses on this section?
For frontmatter stuff like keywords, I think it doesn't hurt to add them as you suggested. But it's also stuff that can more naturally be added once we fill up the pages with content. That's why I left them out for now.

Reply:
I intend this section to cover more than dev env set up, please stick with Install for the section title